test/integration/forms/form.js

```diff
@@ -253,6 +253,100 @@ context('Patient Action Form', function() {
       .should('have.value', 'bar');
   });
 
+  specify('prefill a form with latest submission', function() {
+    cy
+      .server()
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '11111' };
+        fx.data.relationships['program-action'] = { data: { id: '11111' } };
+
+        fx.data.attributes.tags = ['prefill-latest-response'];
+
+        return fx;
+      })
+      .routeFormDefinition()
+      .routeFormActionFields()
+      .routeActionActivity()
+      .routePatientByAction()
+      .routeLatestFormResponseByPatient(fx => {
+        fx.data = {
+          familyHistory: 'Prefilled family history',
+          storyTime: 'Prefilled story time',
+          patient: { fields: { foo: 'bar' } },
+        };
+
+        return fx;
+      })
+      .visit('/patient-action/1/form/11111')
+      .wait('@routeAction')
+      .wait('@routePatientByAction')
+      .wait('@routeFormDefinition')
+      .wait('@routeLatestFormResponseByPatient');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[familyHistory]"]')
+      .should('have.value', 'Prefilled family history');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[storyTime]"]')
+      .should('have.value', 'Prefilled story time');
+
+    cy
+      .iframe()
+      .find('[name="data[patient.fields.foo]"]')
+      .should('have.value', 'bar');
+  });
+
+  specify('prefill a form with latest submission from another form', function() {
+    cy
+      .server()
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '66666' };
+        fx.data.relationships['program-action'] = { data: { id: '11111' } };
+
+        fx.data.attributes.tags = ['prefill-latest-response'];
+
+        return fx;
+      })
+      .routeFormDefinition()
+      .routeFormActionFields()
+      .routeActionActivity()
+      .routePatientByAction()
+      .routeLatestFormResponseByPatient(fx => {
+        fx.data = {
+          familyHistory: 'Prefilled family history',
+          storyTime: 'Prefilled story time',
+          patient: { fields: { foo: 'bar' } },
+        };
+
+        return fx;
+      })
+      .visit('/patient-action/1/form/11111')
```

Comment:
I think this also needs to be `66666`
To test that this test is working after changing this to `66666` if you temporarily remove the `prefill_form_id` from the fixture the test should break.